src/library/commissioner_safe.hpp

```diff
@@ -66,12 +66,15 @@ namespace commissioner {
 class CommissionerSafe : public Commissioner
 {
 public:
-    CommissionerSafe() = default;
+    CommissionerSafe(CommissionerHandler &aHandler)
+        : mHandler(aHandler)
+    {
+    }
 
     CommissionerSafe(const CommissionerSafe &aCommissioner) = delete;
     const CommissionerSafe &operator=(const CommissionerSafe &aCommissioner) = delete;
 
-    Error Init(CommissionerHandler &aHandler, const Config &aConfig);
+    Error Init(const Config &aConfig) override;
```

Comment:
why move `CommissionerHandler &aHandler` out of Init ?

Reply:
Because it has been passed in by the contructor https://github.com/openthread/ot-commissioner/pull/97/files/7242b04c54ae30d3c42c583384a8ab96d7a84ac9#diff-370351476718aeda5c4d8fad87034d0bR69-R72.